@xen-orchestra/proxy/src/app/_decorateResult.js

```diff
@@ -0,0 +1,14 @@
+// @decorateResult(function(result, [arg1, arg2], [fnArg1, fnArg2]) {}, arg1, arg2)
```

Comment:
I don't understand this.
```suggestion
// Applies a series of transformations to the result returned by the method
//
// @decorateResult(transform1, transform2, ...)
// method () {
```

Reply:
How to deal with transform args?